### PR TITLE
Fix typo in "LayerNormalization" docs

### PR DESCRIPTION
**Pull request by Exile333**
### Description
Fix typo in "LayerNormalization" node's description.
Now it is said that attribute "axis" can be equal to "rank(X)", which has no sense. 
If you look in [tests implementation](https://github.com/onnx/onnx/blob/main/onnx/backend/test/case/node/layernormalization.py#L13-L58), you will see that it will produce tensor of zeroes. And [current tests list](https://github.com/onnx/onnx/tree/main/onnx/backend/test/data/node) doesn't include such case. Based on these facts, i assume it is just a typo.

### Motivation and Context
It is a very confusing typo. Me and collegues spent some time trying to figure out, what tensor this case should produce.

### CODE REVIEW

**Comment by yuanyao-nv** [comment]
I think you would need to regenerate the docs as well. See https://github.com/onnx/onnx/blob/main/docs/CONTRIBUTING.md#generated-operator-documentation

File: onnx/defs/nn/defs.cc [hunk 1]
```diff
@@ -2695,7 +2695,7 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetDoc(LayerNormalization_ver17_doc)
         .Attr(
             "axis",
-            "The first normalization dimension. If rank(X) is r, axis' allowed range is [-r, r]. "
+            "The first normalization dimension. If rank(X) is r, axis' allowed range is [-r, r). "
```
Comment (jcwchen) [inline onnx/defs/nn/defs.cc:2698]: It seems that this change is not related to your PR description?

**Review by xadupre — APPROVED** [review]

**Comment by xadupre** [comment]
Operators.md is not updated and the unit tests do not fail. That's unexpected.

**Comment by jcwchen** [comment]
> Operators.md is not updated and the unit tests do not fail. That's unexpected.

https://github.com/onnx/onnx/pull/5367 This PR should fix this issue. Thanks for catching it.

File: onnx/defs/nn/defs.cc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jcwchen) [reply onnx/defs/nn/defs.cc:2698]: Now the enforce-Lint CI can catch the error that the document is not updated. Please use `python onnx/defs/gen_doc.py` to generate Operators.md accordingly. Thanks!

**Comment by gramalingam** [comment]
It seems to me the meaning, if axis=rank, is that the op will behave like an identity operator (no normalization done)? It is not clear to me if this is useful in any edge-case, but at least specifying the meaning seems doable.
